evaluate/repository.go

```diff
@@ -25,7 +25,7 @@ func EvaluateRepository(resultPath string, model model.Model, language language.
 
 	log.Printf("Evaluating model %q using language %q and repository %q", model.ID(), language.ID(), repositoryPath)
 	defer func() {
-		log.Printf("Evaluated model %q using language %q and repository %q: encountered %d problems", model.ID(), language.ID(), repositoryPath, len(problems))
+		log.Printf("Evaluated model %q using language %q and repository %q: encountered %d problems: %+v", model.ID(), language.ID(), repositoryPath, len(problems), problems)
```

Comment:
Here it becomes more verbose?

Reply:
Because that one goes into the log files.

Reply:
I would make all of them verbose. Right now, we don't have log levels, or debugging options. Making debugging harder makes working on bugs harder.

Reply:
okily diokily